README.md

```diff
@@ -109,6 +109,26 @@ class FailedPaymentConsumer
 end
 ```
 
+It is possible to set some custom options to consumer's queue explicitly.
+This example sets the consumer's queue as a
+[quorum queues](https://www.rabbitmq.com/quorum-queues.html)
+and to operate in the [lazy mode](https://www.rabbitmq.com/lazy-queues.html).
+The `initial_group_size` argument is optional.
+
+```ruby
+class FailedPaymentConsumer
+  include Hutch::Consumer
+  consume 'gc.ps.payment.failed'
+  queue_name 'failed_payments'
+  lazy_queue
+  quorum_queue initial_group_size: 5
```

Comment:
Can we please  use  a  lower value? Examples like this tend to be copied straight into production and plastered all over tutorials on the Web, and I'm not sure if everyone wants to run with at least 5 replicas for every queue :)

Reply:
Sure, I will use 3 which is the minimum value that [makes practical sense](https://www.rabbitmq.com/quorum-queues.html#replication-factor)